server/server_test.go

```diff
@@ -120,16 +120,13 @@ func (cli *testServerClient) runTests(t *testing.T, overrider configOverrider, t
 
 // runTestsOnNewDB runs tests using a specified database which will be created before the test and destroyed after the test.
 func (cli *testServerClient) runTestsOnNewDB(t *testing.T, overrider configOverrider, dbName string, tests ...func(dbt *testkit.DBTestKit)) {
-	dsn := cli.getDSN(overrider, func(config *mysql.Config) {
-		config.DBName = ""
-	})
+	dsn := cli.getDSN(overrider)
```

Comment:
Why modify this?

Reply:
```config.DBName = ""``` is useless.

Reply:
whether it has ```config.DBName = ""``` or not, the result will not be changed.

Reply:
recovered